clients/js/src/generated/models/get-embedding.ts

```diff
@@ -32,6 +32,12 @@ export interface GetEmbedding {
      * @memberof GetEmbedding
      */
     'where'?: object;
+    /**
+     * 
+     * @type {object}
+     * @memberof GetEmbedding
+     */
+    'where_document'?: object;
```

Comment:
Object works, but should we add more rigorous typing like what we have in python at some point?

Reply:
this is auto-generated code which comes from here: https://github.com/chroma-core/chroma/blob/ef0007636048042b55303d18c7f4d74265667294/chromadb/server/fastapi/types.py#L39

So if we update these types.. we should get "for free" the JS types

Reply:
Got it makes sense! I can update that end of it.